### ISSUE TITLE
How to get class logits before softmax?

### ISSUE THREAD

**Issue by jackshi081** [facebookresearch/detectron2#2286]
Hi, is there any easy way to get the class logits before softmax (e.g through Default predictor)? I am simply inputing RGB images for a detection task using Faster RCNN provides in detectron2

**Comment by ppwwyyxx** [comment]
see https://detectron2.readthedocs.io/tutorials/models.html#partially-execute-a-model
